README.md

```diff
@@ -1,2 +1,120 @@
+| `email`          | The email address of your Firebolt account.                                                                                                           | **true**  | email@test.com       |
+| `password`       | The password of your Firebolt account.                                                                                                                | **true**  | password             |
+| `accountName`    | The account name of your Firebolt account.                                                                                                            | **true**  | `super_organization` |
+| `engineName`     | The engine name of your Firebolt engine.                                                                                                              | **true**  | `my_super_engine`    |
+| `db`             | The name of your database.                                                                                                                            | **true**  | test                 |
+| `table`          | The name of a table in the database that the connector should read from, by default.                                                                  | **true**  | clients              |
+| `orderingColumn` | Column which is using for ordering in select query. It can be a pk or a timestamp column                                                              | **true**  | created_date         |
+| `columns`        | Comma separated list of column names that should be included in the each Record's payload. By default: all columns.                                   | **false** | "id,name,age"        |
+| `primaryKey`     | The name of the column that records should use for their `key` fields.                                                                                          | **true**  | "id"                 |
+| `batchSize`      | Size of batch. By default is 100. <b>Important:</b> Please, don’t update this variable after running the pipeline, as this will cause position issues. | **false** | "100"                |
+
+### Snapshot iterator
+
+The snapshot iterator starts getting data from the table using post request with select query with limit and offset and
+ordering by `orderingColumn`. For example `select * from {table} order by {orderingColumn} limit 20 offset 0`. Batch size
+is configurable, offset value is zero for the first time. Iterator saves rows from table to `currentBatch` slice variable.
+The Iterator `HasNext` method checks if the next element exists in the currentBatch using variable `index` and,
+if necessary, changes the `offset` and runs a select query to get a new data with the new offset. Method `Next` 
+gets next element and converts it to `Record` sets metadata variable table.
+Set record metadata variable `action` - `insert`, increases `index`.
```

Comment:
We can't use tracking table because firebolt doesn't have triggers and update and delete operations. And you right ordered select not needed for today. I think  when firebolt add functionality (deleting, updating), we have orderingColumn and it will be easier expand code. But I can remove logic with ordering, what do you think?

Reply:
IMHO, if we are sure that we will be using ordering once/if Firebolt adds this functionality, then I agree. But it looks like we're not sure if or when Firebolt will be adding this functionality, and how will it look like, so at least for now I think we should keep it as simple as possible.

Reply:
ok I will simplify it

Reply:
simplified